src/buildHTML.js

```diff
@@ -1232,6 +1227,18 @@ groupTypes.delimsizing = function(group, options) {
             [group.value.mclass]);
 };
 
+groupTypes.verb = function(group, options) {
+    var text = buildCommon.makeVerb(group, options);
+    var body = [];
+    var i;
+    for (i = 0; i < text.length; i++) {
+        body.push(buildCommon.makeSymbol(
+            text[i], "Main-Regular", group.mode, options, ["mathtt"]));
+    }
+    buildCommon.tryCombineChars(body);
+    return makeSpan(["mord", "verb"], body, options);
```

Comment:
Why the `"verb"` class? I don't see any specific styling for this. And the more classes we use in KaTeX, the more classes may not be usable by the surrounding page for their own purpose. Personally I'd even suggest prefixing all our classes with `KaTeX-` or some such, in order to avoid conflicts. Unless we do that, I'd be extra careful to only introduce new classes when we have to.